pkg/skaffold/graph/dependencies_test.go

```diff
@@ -55,3 +58,41 @@ func TestSourceDependenciesCache(t *testing.T) {
 		}
 	})
 }
+
+func Test_sourceDependenciesForArtifact(t *testing.T) {
```

Comment:
And we'd use `TestSourceDependenciesForArtifact`

Reply:
Done